lib/test_suite.py

```diff
@@ -110,6 +111,14 @@ def __init__(self, suite_path, args):
                 lambda x: os.path.join(suite_path, x),
                 dict.fromkeys(self.ini[i].split())
                 if i in self.ini else dict())
+        for item in config.items("default"):
+            if item[0] == "fragile":
+                try:
+                    self.fragile = json.loads(item[1])
+                except ValueError:
+                    print "WARNING: suite '{}' uses old fragile format" . format(suite_path)
```

Comment:
test-run redirects standard input / output / error streams and so `color_stdout()` is used across the whole codebase. This print will likely excuted before the redirection, but I would stay on the safe side and use `color_stdout()` as usual. BTW, it prints everything line-by-line and so prevent mixing output of different processes in a strange way.

I would accept the old format without deprecation warnings, but it is up to you.

Reply:
Right, decided to remove.